projects/1.18/assets/every-compat/everycomp/lang/zh_cn.json

```
@@ -791,6 +791,33 @@
   "leaves_type.terraqueous.peach": "桃树",
   "leaves_type.terraqueous.pear": "梨树",
   "leaves_type.terraqueous.plum": "李树",
+  "wood_type.terrestria.cypress": "柏木",
+  "wood_type.terrestria.hemlock": "铁杉木",
+  "wood_type.terrestria.japanese_maple": "日本枫木",
+  "wood_type.terrestria.rainbow_eucalyptus": "彩虹桉木",
+  "wood_type.terrestria.redwood": "红杉木",
+  "wood_type.terrestria.rubber": "橡胶木",
+  "wood_type.terrestria.sakura": "樱花木",
+  "wood_type.terrestria.willow": "柳木",
+  "wood_type.terrestria.yucca_palm": "丝兰棕榈木",
+  "leaves_type.terrestria.cypress": "柏树",
+  "leaves_type.terrestria.hemlock": "铁杉",
+  "leaves_type.terrestria.japanese_maple": "日本枫",
+  "leaves_type.terrestria.rainbow_eucalyptus": "彩虹桉",
+  "leaves_type.terrestria.redwood": "红杉",
+  "leaves_type.terrestria.rubber": "橡胶",
+  "leaves_type.terrestria.sakura": "樱花",
+  "leaves_type.terrestria.willow": "柳树",
+  "leaves_type.terrestria.yucca_palm": "丝兰棕榈",
+  "leaves_type.terrestria.jungle_palm": "丛林棕榈",
+  "leaves_type.terrestria.dark_japanese_maple": "日本红枫",

```

```suggestion
  "leaves_type.terrestria.dark_japanese_maple": "暗色日本红枫",
```
那就按你的来吧。我想的是深色，但是怕深色有“深色橡树”的误导向。暗色也许确实好一点？